src/main/java/appeng/client/render/effects/LightningFX.java

```diff
@@ -31,7 +31,6 @@
 
 public class LightningFX extends EntityFX
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Override this superclass' "equals" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2160)